P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -2608,6 +2608,33 @@ effective than the free form <gi>note</gi> element; these elements are
 available when the module for simple analysis is selected (see section
 <ptr target="#AISP"/>).
  </p>
+
+ <p>When alternative versions of the same note, e.g. in different languages or
+   presenting a short form alongside a more elaborate form of a note it is possible to group
+   such notes within the <gi>noteGrp</gi> element. Such notes would make use of
+   attributes such as <att>xml:lang</att> or <att>type</att> to differentiate their
```

Comment:
Two thoughts: first, to be more specific about which notes we’re talking about; and second, wouldn’t the common `@target` typically be on the parent `<noteGrp>`, i.e. inherited by all the child `<note>`s?

If so, perhaps tweaking the wording to something like:
~~~xml
Typically the <gi>note</gi> elements within a <gi>noteGrp</gi> would
be differentiated by use of attributes such as <att>xml:lang</att> or
<att>type</att>, while sharing the same point of attachment (either
implicitly in case of inline notes, or explicitly specified via a
<att>target</att> attribute, which may be specified on the
<gi>noteGrp</gi> itself).</p>
~~~

Reply:
agreed, thank you